docs/intermediate/tutorial11-normals/README.md

```diff
@@ -279,6 +279,9 @@ impl Model {
                 let r = 1.0 / (delta_uv1.x * delta_uv2.y - delta_uv1.y * delta_uv2.x);
                 let tangent = (delta_pos1 * delta_uv2.y - delta_pos2 * delta_uv1.y) * r;
                 let bitangent = (delta_pos2 * delta_uv1.x - delta_pos1 * delta_uv2.x) * r;
+                // We flip the bitangent to enable right-handed normal
+                // maps with wgpu texture coordinate system
+                let flipped_bitangent = bitangent * -1.0;
```

Comment:
I'd probably just add the -1.0 to the `let bitangent = ...` line instead, and then add a comment above stating that we're using a right handed coordinate system.

Alternatively you could also flip the subtraction. Something like:
```rust
let bitangent = (delta_pos1 * delta_uv2.x - delta_pos2 * delta_uv1.y) * r;
```

Reply:
@sotrh updated.